index.html

```diff
@@ -45,7 +45,7 @@ <h2 data-i18n="app.title"></h2>
                             <input class='col8' id='address' placeholder='' data-i18n="[placeholder]step1.addr"/>
                             <button class='col4' id='findme' data-i18n="step1.button"></button>
                             <br />
-                            Or <a id="use_my_location" href="#" onclick="return false">use my current location</a>.
+                            <a id="use_my_location" data-i18n="step1.use_location" href="#" onclick="return false">use my current location</a>.
```

Comment:
I am thinking a button like [this](https://cdn3.iconfinder.com/data/icons/glypho-travel/64/gps-position-target-512.png) between the field and the current button. The translation would be for a tooltip, without the "or".